From 867c97fb67caa40cf1d42aa52639152390f5870d Mon Sep 17 00:00:00 2001
From: Yong Zhao <yong.zhao@amd.com>
Date: Wed, 30 Mar 2016 18:26:03 -0400
Subject: [PATCH 1512/1722] SDMA macro define missing.

This define should be provided from the IP block header. Add it
directly to the source file as a quick fix.

Change-Id: I215d37f9842015ac7f376abeb8cc118f94b7e840
---
 drivers/gpu/drm/amd/amdgpu/gfx_v7_0.c | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/drivers/gpu/drm/amd/amdgpu/gfx_v7_0.c b/drivers/gpu/drm/amd/amdgpu/gfx_v7_0.c
index bec58a4..49c5bb9 100644
--- a/drivers/gpu/drm/amd/amdgpu/gfx_v7_0.c
+++ b/drivers/gpu/drm/amd/amdgpu/gfx_v7_0.c
@@ -47,6 +47,8 @@
 #include "oss/oss_2_0_d.h"
 #include "oss/oss_2_0_sh_mask.h"
 
+#define NUM_SIMD_PER_CU 0x4
+
 #define GFX7_NUM_GFX_RINGS     1
 #define GFX7_NUM_COMPUTE_RINGS 8
 
-- 
2.7.4

